python/lsst/ts/ess/common/test_utils.py

```diff
@@ -100,6 +105,14 @@ def check_hx85ba_reply(
                     assert common.device.MockPressureConfig.min <= resp[i]
                     assert resp[i] <= common.device.MockPressureConfig.max
 
+        dew_point = common.sensor.Hx85baSensor.compute_dew_point(
+            relative_humidity=resp[0], temperature=resp[1]
+        )
+        if math.isnan(resp[0]) or math.isnan(resp[1]):
+            assert math.isnan(resp[3])
+        else:
+            assert resp[3] == pytest.approx(dew_point)
```

Comment:
Note that pytest.approx has a `nan_ok` float, so I think you can just write:
```
assert resp[3] == pytest.approx(dew_point, nan_ok=True)
```

Reply:
Works wonderfully well, thanks!